made dice.cs

kevyonan opened the pull request:
let's users play with dice by using the command: "!roll d20" to roll a 20 sided dice.
available sizes are 6, 10, 20, 50, 100 sided die.

Commit:
made dice.cs

let's users play with dice by using the command: "!roll d20" to roll a 20 sided dice.
available sizes are 6, 10, 20, 50, 100 sided die.

Headline commented on Commands/Dice.cs:
```diff
@@ -0,0 +1,35 @@
+
+namespace ChancyBot.Commands
+{
+	public class HelpCommand : ModuleBase
```
HelpCommand should be DiceCommand

Headline commented on Commands/Dice.cs:
```diff
@@ -0,0 +1,35 @@
+		{
+			Random rand = new Random();
+			int num = 0;
+			if( arg == "d6" )
```
We should probably be comparing strings with .Equals

Commit:
making changes headline wants

changed string comparators to use the 'Equals' method.
changed class name to avoid name conflicts.

kevyonan commented:
done!

Headline committed:
Add Compile Include
